src/pages/Main/SlideButton/SlideButton.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './SlideButton.scss';
+
+class SlideButton extends React.Component {
+  render() {
+    return (
+      <>
+        <div
+          onClick={this.props.click}
+          className={this.props.rightButton ? 'clickRight' : 'clickRightHidden'}
+        >
+          <i className="fas fa-chevron-right"></i>
+        </div>
+        <div
+          onClick={this.props.click}
+          className={this.props.rightButton ? 'clickLeftHidden' : 'clickLeft'}
+        >
+          <i className="fas fa-chevron-left"></i>
+        </div>
```

Comment:
hidden이 아닌 상태일 때, 리스트 컨테이너의 아랫쪽에 자리를 차지하고 있네요. 이렇게 되면 나중에 레이아웃을 수정할 일이 생기면 유지보수가 어려울 수 있습니다.
또한, 명칭 관련해서 위의 리뷰처럼 혼란을 줄 수 있어 다음과 같이 변경해볼 수 있을 것 같습니다.
```suggestion
        <div
          onClick={click}
          className={`rightBtn ${isLeftBtn ? 'hidden' : ''}`}
        >
          <i className="fas fa-chevron-right" />
        </div>
        <div onClick={click} className={`leftBtn ${isLeftBtn ? '' : 'hidden'}`}>
          <i className="fas fa-chevron-left" />
        </div>
```

Reply:
아 감사합니다!
해당 부분을 잠시 잊고 있었는데
덕분에 훨씬 가독성이나 유지보수 측면에서 도움이 됐습니다!